definitions.yml

```
@@ -973,6 +973,16 @@ definitions:
       wallet_activation_disabled:
         description: "Flag that disables the button to activate the wallet"
         type: boolean
+      deferred_issuance_mdl:
```

We're currently using this data to display a custom message for the deferred issuance of the MDL credential. However, are we certain this won't be needed for other credentials in the future?
Additionally, to make its purpose clearer, I suggest adding more context to its name. For example,  something like `*_screen_content`. 

```suggestion
      deferred_issuance_screen_content:
```

---

Yes, you're right! Thanks for the comment